### PR TITLE
Introduce a Related Resources section for n-tier-content, adding some additional links

### PR DESCRIPTION
**Pull request by udidahan**
Hope this works @ckittel 

  

  
<details><summary>AFTER YOUR PR HAS BEEN CREATED, expand this section for tips and additional instructions.</summary>    
      

These are common guidelines for contributions across the repos managed by the Cloud Architecture Content Team (CACT). Some repositories may have additional specific requirements that are not listed here.   

## Guidance for all contributors  
  
  | **Topic** | **Guidance** |
  | ----------| ------------ |
  | **Draft PR** | If your PR will be a work-in-progress for more than a day or two, select the **Convert to draft** link in the upper right of the page (under **Reviewers**) to change it to a draft. For future reference, you can also do this using the **Create pull request** button drop-down during PR creation. | 
  | **ms.date metadata** | <ul><li>Don't update an article's "ms.date" metadata property unless you've done a **full freshness review** of the content. A full freshness review includes changes required to correct or improve the **full** technical accuracy of the article.</li><li>Don't update "ms.date" if you're doing targeted changes to improve non-technical aspects of the article, such as editorial quality, art improvements, article template alignment, etc.</li><li>If you've changed any "ms.date" properties for work that wasn't part of full review for freshness, please reset them to their previous value.</li></ul> | 
  | **Placement and linking** | If you're creating a new article or articles, include updates to the related TOC.yml file to propose where the article(s) should be placed. Also consider other places within the document set where it would be beneficial to cross-reference and link to your new article(s). | 
  | **PR build** | After you open your PR, and for each successive commit that you push to your branch, the publishing platform will run validation on the files in your pull request. A summary of the build results for each file will be inserted inline into your pull request, which includes any build suggestions/warnings/errors. PRs cannot be merged until all build errors and most warnings are resolved. |
  | **Publishing** | Following a successful merge, most repos publish to the live site at least once per (business) day, usually around 10am Pacific. |
  | **Additional resources** | <ul><li>[Learn.Microsoft.Com contributor guide](https://review.learn.microsoft.com/help/contribute/?branch=main)</li></ul>
  
## Additional guidance for private repos and internal contributors  

  | **Topic** | **Guidance** | 
  | ----------| ------------ | 
  | **PR size** | If your PR is more than ~5 lines of changes, or you'd like for the changes to go through editorial or larger review, open a contribution request at https://aka.ms/Contribution and include a link to the PR in response #8. Once it's processed, you'll be notified of the next steps.  |
  | **PR title prefix** | Select the **Edit** button to the right of the PR title if you need to revise it. The following prefixes are reserved for specific contribution types:<br/><br/><ul><li>**[Quality Check]** - maintenance work related to content quality (edit passes, art improvements, template alignment)</li><li>**[LinkFix]** - recurring/adhoc PRs to correct link URLs</li><li>**[Pipeline]** - new/updated contributor success pipeline content</li><li>**[WIP]** - a work-in-progress draft requiring several days/weeks</li></ul> |
  | **PR preview** | Following successful build of your PR, publishable files will also include **Preview URL** links to staged previews of your new/updated articles. Be sure to review these for verification of your intended contributions, or to send to other internal contributors for review. |
  | **PR sign-off (public repo)** | If an article you own is updated in a public repo PR, you are responsible for sign-off. You will be automatically notified via email. The PR will not be merged until you've had a chance to review and sign-off. |
  | **PR sign-off (private repo)** | After you've completed your proposed changes, addressed build warnings, and completed all review work, you can begin the sign-off process for review and merge:<br/><br/><ol><li>If your PR is in draft mode, remove "[WIP]" from the title and select **Ready for review** button at the bottom of the PR.</li><li>Enter "#sign-off" in a new comment. This comment indicates that **you're confident the work meets or exceeds Microsoft's standards for publication**, and will trigger the review process.</li><li>Your PR may be selected for initial review by the CACT. Following CACT review, you may receive questions or requests for additional changes. You should have initial feedback from CACT review within a few business days. If you have an urgent request or need to contact the team, please mention `@MicrosoftDocs/cloud-architecture-content-team-pr-reviewers` in your PR and someone will get back to you. After CACT review is complete, a `CACT #sign-off` will be added.</li><li>Final review/merge is done by the PR review team. The PR team may also respond with feedback, categorized as "Blocking" (requires action from you), or "Non-blocking" (to be addressed in a future PR).</li></ol> |
  | **Additional resources** | <ul><li>[Learn.Microsoft.Com internal contributor guide](https://review.learn.microsoft.com/help/contribute/?branch=main)</li><li>Authoring templates: [architecture-center-pr](https://review.learn.microsoft.com/help/contribute/architecture-center/templates/sample-solution-templates?branch=main), [well-architected-pr](https://review.learn.microsoft.com/help/contribute/global-waf-template?branch=main)</li><li>To contact the CACT use [e-mail](mailto:cact-pr-reviewers@microsoft.com?subject=Help%20with%20pull%20request), or @mention our GitHub team in your PR comments using: `@MicrosoftDocs/cloud-architecture-content-team-pr-reviewers`</li></ul>
</details>

### CODE REVIEW

File: docs/guide/architecture-styles/n-tier-content.md [hunk 1]
```diff
@@ -68,26 +68,20 @@ Network security groups restrict access to each tier. For example, the database
 > [!NOTE]
 > The layer labeled "Business Tier" in our reference diagram is a moniker to the business logic tier. Likewise, we also call the presentation tier the "Web Tier." In our example, this is a web application, though multi-tier architectures can be used for other topologies as well (like desktop apps).  Name your tiers what works best for your team to communicate the intent of that logical and/or physical tier in your application - you could even express that naming in resources you choose to represent that tier (e.g. vmss-appName-business-layer).
 
-For more information about running N-tier applications on Azure:
-
-- [N-tier application with Apache Cassandra][n-tier-linux]
-- [Windows N-tier application on Azure with SQL Server][n-tier-windows-SQL]
-- [Microsoft Learn module: Tour the N-tier architecture style](/training/modules/n-tier-architecture/)
-- [Azure Bastion](/azure/bastion/bastion-overview)
```
Comment (udidahan) [inline docs/guide/architecture-styles/n-tier-content.md:76]: Moved to the Related Resources section below.

File: docs/guide/architecture-styles/n-tier-content.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ An N-tier architecture divides an application into **logical layers** and **phys
 
 Layers are a way to separate responsibilities and manage dependencies. Each layer has a specific responsibility. A higher layer can use services in a lower layer, but not the other way around.
 
-Tiers are physically separated, running on separate machines. A tier can call to another tier directly, or use asynchronous messaging (message queue). Although each layer might be hosted in its own tier, that's not required. Several layers might be hosted on the same tier. Physically separating the tiers improves scalability and resiliency, but also adds latency from the additional network communication.
+Tiers are physically separated, running on separate machines. A tier can call to another tier directly, or use [asynchronous messaging (message queue)](/azure/service-bus-messaging/service-bus-async-messaging). Although each layer might be hosted in its own tier, that's not required. Several layers might be hosted on the same tier. Physically separating the tiers improves scalability and resiliency, but also adds latency from the additional network communication.
```
Comment (ckittel) [inline docs/guide/architecture-styles/n-tier-content.md:7]:
```suggestion
Tiers are physically separated, running on separate machines. A tier can call to another tier directly, or use [Asynchronous messaging patterns](/azure/service-bus-messaging/service-bus-async-messaging) through a message queue. Although each layer might be hosted in its own tier, that's not required. Several layers might be hosted on the same tier. Physically separating the tiers improves scalability and resiliency, but also adds latency from the additional network communication.
```

File: docs/guide/architecture-styles/n-tier-content.md [hunk 3]
```diff
@@ -68,26 +68,20 @@ Network security groups restrict access to each tier. For example, the database
...
+- Use virtual machine scale sets for [autoscaling][auto-scaling].
 
 - Look for places in the architecture where you can use a managed service without significant refactoring. In particular, look at caching, messaging, storage, and databases.
 
 - For higher security, place a network DMZ in front of the application. The DMZ includes network virtual appliances (NVAs) that implement security functionality such as firewalls and packet inspection. For more information, see [Network DMZ reference architecture][dmz].
 
-- For high availability, place two or more NVAs in an availability set, with an external load balancer to distribute Internet requests across the instances. For more information, see [Deploy highly available network virtual appliances][ha-nva].
+- For high availability, place two or more NVAs in an availability set, with an external load balancer to distribute Internet requests across the instances. For more information, see [Deploy highly available network virtual appliances][ha-nva] and this example scenario of [High availability and disaster recovery for IaaS apps](/azure/architecture/example-scenario/infrastructure/iaas-high-availability-disaster-recovery)
```
Comment (ckittel) [inline docs/guide/architecture-styles/n-tier-content.md:84]:
```suggestion
- For high availability, place two or more NVAs in an availability set, with an external load balancer to distribute Internet requests across the instances. For more information, see [Deploy highly available network virtual appliances][ha-nva] and this example scenario of [High availability and disaster recovery for IaaS apps](/azure/architecture/example-scenario/infrastructure/iaas-high-availability-disaster-recovery).
```

File: docs/guide/architecture-styles/n-tier-content.md [hunk 4]
```diff
@@ -97,6 +91,15 @@ For more information about running N-tier applications on Azure:
 
 - If you need higher availability than the Azure SLA for VMs provides, replicate the application across two regions and use Azure Traffic Manager for failover. For more information, see [Run Windows VMs in multiple regions][multiregion-windows] or [Run Linux VMs in multiple regions][multiregion-linux].
 
+### Related Resources
+
+- [N-tier application with Apache Cassandra][n-tier-linux]
+- [Windows N-tier application on Azure with SQL Server][n-tier-windows-SQL]
+- [Microsoft Learn module: Tour the N-tier architecture style](/training/modules/n-tier-architecture/)
+- [Azure Bastion](/azure/bastion/bastion-overview)
+- More information on using [messaging in N-Tier architectures](https://docs.particular.net/architecture/azure/n-tier)
```
Comment (ckittel) [inline docs/guide/architecture-styles/n-tier-content.md:100]:
```suggestion
- More information on messaging in an [N-tier architecture style on Azure](https://docs.particular.net/architecture/azure/n-tier)
```

File: docs/guide/architecture-styles/n-tier-content.md [hunk 5]
```diff
@@ -97,6 +91,15 @@ For more information about running N-tier applications on Azure:
...
+
+
```
Comment (ckittel) [inline docs/guide/architecture-styles/n-tier-content.md:102]:
```suggestion
```

**Review by ckittel — CHANGES_REQUESTED** [review]

**Review by ckittel — DISMISSED** [review]

**Comment by ckittel** [comment]
CACT #sign-off

#remove-label:"cact-review-in-progress"
#remove-label:"do-not-merge"
#label:"ready-to-merge"

**Comment by udidahan** [comment]
Wonderful! Thanks very much :)

File: docs/guide/architecture-styles/n-tier-content.md [hunk 6]
```diff
@@ -68,26 +68,20 @@ Network security groups restrict access to each tier. For example, the database
...
-- [Azure Bastion](/azure/bastion/bastion-overview)
 
 ### Additional considerations
 
 - N-tier architectures are not restricted to three tiers. For more complex applications, it is common to have more tiers. In that case, consider using layer-7 routing to route requests to a particular tier.
 
 - Tiers are the boundary of scalability, reliability, and security. Consider having separate tiers for services with different requirements in those areas.
 
-- Use virtual machine scale sets for autoscaling.
+- Use virtual machine scale sets for [autoscaling][auto-scaling].
```
Comment (Court72) [inline docs/guide/architecture-styles/n-tier-content.md:78]:
```suggestion
- Use virtual machine scale sets for [autoscaling][autoscaling].
```

**Review by Court72 — APPROVED** [review]

**Pull request merged**